### DISCUSSION TITLE
How should I add search string searx.be?

### DISCUSSION THREAD

**Discussion by DragonSerW** [getpelican/pelican#3202] [Ideas]
Hello guys!
I personally create web-site splash.org.ru, where I place internet-links, and I install my website as start page into Firefox and Chromium browsers to people who ask me to tune those people mobile phones and desktops.
So my question is: how can I add SearXNG search field to my web-site?
I do not need search engine for my site, but I need Internet Search string :-))

**Comment by justinmayer** [comment]
Hi @DragonSerW. I don't really understand this question. Also, I don't understand what it has to do with Pelican?

**Reply by DragonSerW** [reply]
The engine of splash.org.ru is Pelican.
Can I write my question in Russian language? :)

**Reply by justinmayer** [reply]
I don't understand Russian, so I don't think writing your question in Russian will help. Pelican generates HTML from Markdown / reStructuredText, so it is not clear to me what this question has to do with Pelican specifically.

**Comment by jorgesumle** [comment]
Puedes crear un formulario que te lleve a una URL de búsqueda de ese buscador:

```
<form action="https://searx.be/search" method="get">
<input name="q" type="text" value="">
<button>Buscar</button>
</form>
```

No sé si es esto lo que preguntas.
